src/main/groovy/net/vivin/gradle/versioning/tasks/TagTask.groovy

```diff
@@ -13,11 +14,24 @@ import org.gradle.tooling.BuildException
  * @author vivin
  */
 class TagTask extends DefaultTask {
+
+    Closure<String> tagMessage = {
+        "v${project.version}"
+    }
+
+    private Closure<String> fromSystemProperyClosure = { String p ->
```

Comment:
Besides the typo,
I added an additional one for environment variables.
I really like the currying approach.
I always forget about the currying and do it in other ways where it would be possible.